python/ray/monitor.py

```diff
@@ -36,6 +36,9 @@
 # xray heartbeats
 XRAY_HEARTBEAT_CHANNEL = b"6"
 
+# xray driver updates
+XRAY_DRIVER_CHANNEL = b"8"
```

Comment:
this should go in `gcs_utils.py` (same with `XRAY_HEARTBEAT_CHANNEL`)

Actually, it should be `ray.gcs_utils.TablePubsub.DRIVER`